[question]: Problem build project Xamarin iOS (vs Mac 2022) with OnesignalSdk.DotNet

### How can we help?

My xamarin ios project with xcode 14.3 and visual studio mac 2022, if I use OnesignalSdk.Dotnet when I compile the project I get the following errors:

    Process exited with code 1, command:
    /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/clang -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalFramework.xcframework/ios-arm64_x86_64-simulator -framework OneSignalFramework -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalCore.xcframework/ios-arm64_x86_64-simulator -framework OneSignalCore -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalExtension.xcframework/ios-arm64_x86_64-simulator -framework OneSignalExtension -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalNotifications.xcframework/ios-arm64_x86_64-simulator -framework OneSignalNotifications -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalOSCore.xcframework/ios-arm64_x86_64-simulator -framework OneSignalOSCore -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalOutcomes.xcframework/ios-arm64_x86_64-simulator -framework OneSignalOutcomes -F /Users/fabiocirillo/.nuget/packages/onesignalsdk.dotnet/5.0.0/res/ios/OneSignalUser.xcframework/ios-arm64_x86_64-simulator -framework OneSignalUser -framework CoreFoundation -framework Security -framework UserNotificationsUI -framework WatchConnectivity -framework Speech -framework SafariServices -framework QuickLook -framework PushKit -framework PhotosUI -framework NotificationCenter -framework NewsstandKit -framework Messages -framework LocalAuthentication -framework IntentsUI -framework IdentityLookup -framework HealthKitUI -framework FileProviderUI -framework ExternalAccessory -framework DeviceCheck -framework CoreAudioKit -framework ContactsUI -framework iAd -framework Vision -framework VideoToolbox -framework VideoSubscriberAccount -framework UserNotifications -framework Twitter -framework SystemConfiguration -framework StoreKit -framework SpriteKit -framework Social -framework SceneKit -framework ReplayKit -framework Photos -framework PDFKit -framework PassKit -framework OpenGLES -framework NetworkExtension -framework MultipeerConnectivity -framework ModelIO -framework MetalKit -framework MessageUI -framework MediaToolbox -framework MediaPlayer -framework MediaAccessibility -framework MapKit -framework JavaScriptCore -framework Intents -framework ImageIO -framework HomeKit -framework HealthKit -framework GameplayKit -framework GameKit -framework GameController -framework GLKit -framework WebKit -framework Foundation -framework FileProvider -framework EventKitUI -framework EventKit -framework CoreVideo -framework CoreText -framework CoreTelephony -framework CoreSpotlight -framework CoreMotion -framework CoreMIDI -framework CoreMedia -framework CoreML -framework CoreLocation -framework CoreImage -framework CoreGraphics -framework CoreData -framework CoreBluetooth -framework QuartzCore -framework Contacts -framework CloudKit -framework CallKit -framework AudioToolbox -framework AssetsLibrary -framework UIKit -framework AddressBookUI -framework AddressBook -framework AdSupport -framework Accounts -framework Accelerate -framework AVKit -framework AVFoundation -framework ARKit -framework MetalPerformanceShaders -framework Metal -framework CFNetwork -framework GSS -weak_framework VisionKit -weak_framework UniformTypeIdentifiers -weak_framework SoundAnalysis -weak_framework SharedWithYou -weak_framework ScreenTime -weak_framework QuickLookThumbnailing -weak_framework PencilKit -weak_framework OSLog -weak_framework LinkPresentation -weak_framework IdentityLookupUI -weak_framework CoreLocationUI -weak_framework CoreHaptics -weak_framework BackgroundTasks -weak_framework AutomaticAssessmentConfiguration -weak_framework AppTrackingTransparency -weak_framework AppClip -weak_framework AdServices -weak_framework SharedWithYouCore -weak_framework SensorKit -weak_framework PushToTalk -weak_framework Network -weak_framework NearbyInteraction -weak_framework NaturalLanguage -weak_framework MetricKit -weak_framework MetalPerformanceShadersGraph -weak_framework ClassKit -weak_framework CarPlay -weak_framework MobileCoreServices -weak_framework BusinessChat -weak_framework BackgroundAssets -weak_framework AuthenticationServices -weak_framework Accessibility -weak_framework AVRouting -weak_framework SystemConfiguration -weak_framework UserNotifications -weak_framework WebKit -weak_framework CoreGraphics -weak_framework UIKit -weak_framework OneSignalOutcomes -weak_framework OneSignalCore -weak_framework OneSignalExtension -weak_framework OneSignalNotifications -weak_framework OneSignalOSCore -weak_framework OneSignalUser -weak_framework OpenAL -weak_framework OneSignalOutcomes -weak_framework OneSignalCore -weak_framework OneSignalExtension -weak_framework OneSignalNotifications -weak_framework OneSignalOSCore -weak_framework OneSignalUser -Xlinker -rpath -Xlinker @executable_path/Frameworks /Library/Frameworks/Xamarin.iOS.framework/Versions/16.4.0.6/SDKs/MonoTouch.iphonesimulator.sdk/lib/Xamarin.iOS.registrar.a /Users/fabiocirillo/Desktop/FinimpresaApp/FinimpresaApp/FinimpresaApp.iOS/obj/iPhoneSimulator/Debug/mtouch-cache/x86_64/main.o /Library/Frameworks/Xamarin.iOS.framework/Versions/16.4.0.6/SDKs/MonoTouch.iphonesimulator.sdk/lib/libmonosgen-2.0.a /Library/Frameworks/Xamarin.iOS.framework/Versions/16.4.0.6/SDKs/MonoTouch.iphonesimulator.sdk/lib/libxamarin-debug.a /Library/Frameworks/Xamarin.iOS.framework/Versions/16.4.0.6/SDKs/MonoTouch.iphonesimulator.sdk/lib/libmono-native-unified.a -force_load /Library/Frameworks/Xamarin.iOS.framework/Versions/16.4.0.6/SDKs/MonoTouch.iphonesimulator.sdk/lib/libapp.a -lcompression -Wl,-pie -arch x86_64 -gdwarf-2 -std=c++14 -I/Library/Frameworks/Xamarin.iOS.framework/Versions/16.4.0.6/SDKs/MonoTouch.iphonesimulator.sdk/include -isysroot /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator16.4.sdk -Qunused-arguments -fobjc-legacy-dispatch -fobjc-abi-version=2 -mios-simulator-version-min=11.0 -lz -liconv -lz -o /Users/fabiocirillo/Desktop/FinimpresaApp/FinimpresaApp/FinimpresaApp.iOS/bin/iPhoneSimulator/Debug/FinimpresaApp.iOS.app/FinimpresaApp.iOS -u _SystemNative_ConvertErrorPlatformToPal -u _SystemNative_ConvertErrorPalToPlatform -u _SystemNative_StrErrorR -u _SystemNative_GetNonCryptographicallySecureRandomBytes -u _SystemNative_GetReadDirRBufferSize -u _SystemNative_ReadLink -u _SystemNative_FStat2 -u _SystemNative_Stat2 -u _SystemNative_LStat2 -u _SystemNative_Symlink -u _SystemNative_ChMod -u _SystemNative_CopyFile -u _SystemNative_GetEGid -u _SystemNative_GetEUid -u _SystemNative_LChflags -u _SystemNative_LChflagsCanSetHiddenFlag -u _SystemNative_Link -u _SystemNative_MkDir -u _SystemNative_Rename -u _SystemNative_RmDir -u _SystemNative_UTime -u _SystemNative_UTimes -u _SystemNative_Unlink -u _SystemNative_OpenDir -u _SystemNative_ReadDirR -u _SystemNative_CloseDir -u _xamarin_timezone_get_local_name -u _xamarin_timezone_get_names -u _xamarin_timezone_get_data -u _xamarin_GetFolderPath -u _xamarin_log -u _xamarin_get_locale_country_code -u _xamarin_CGPoint__VNNormalizedFaceBoundingBoxPointForLandmarkPoint_Vector2_CGRect_nuint_nuint_string -u _xamarin_CGPoint__VNImagePointForFaceLandmarkPoint_Vector2_CGRect_nuint_nuint_string -u _xamarin_release_managed_ref -u _xamarin_set_gchandle_with_flags_safe -u _xamarin_localized_string_format -u _xamarin_localized_string_format_1 -u _xamarin_localized_string_format_2 -u _xamarin_localized_string_format_3 -u _xamarin_localized_string_format_4 -u _xamarin_localized_string_format_5 -u _xamarin_localized_string_format_6 -u _xamarin_localized_string_format_7 -u _xamarin_localized_string_format_8 -u _xamarin_localized_string_format_9 -u _xamarin_init_nsthread -u _xamarin_os_log -u _xamarin_UIApplicationMain -u _xamarin_find_protocol_wrapper_type -u _xamarin_release_block_on_main_thread -u _xamarin_get_original_working_directory_path -u _xamarin_start_wwan -u _xamarin_get_block_descriptor -u _xamarin_simd__NMatrix4_objc_msgSend -u _xamarin_simd__NMatrix4_objc_msgSendSuper -u _xamarin_simd__NMatrix4_objc_msgSend_stret -u _xamarin_simd__NMatrix4_objc_msgSendSuper_stret -u _xamarin_simd__IntPtr_objc_msgSend_NMatrix4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_NMatrix4 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_NMatrix4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_NMatrix4 -u _xamarin_simd__NVector3_objc_msgSend -u _xamarin_simd__NVector3_objc_msgSendSuper -u _xamarin_simd__NVector3_objc_msgSend_stret -u _xamarin_simd__NVector3_objc_msgSendSuper_stret -u _xamarin_simd__NMatrix3_objc_msgSend -u _xamarin_simd__NMatrix3_objc_msgSendSuper -u _xamarin_simd__NMatrix3_objc_msgSend_stret -u _xamarin_simd__NMatrix3_objc_msgSendSuper_stret -u _xamarin_simd__CGPoint_objc_msgSend_NVector3_IntPtr_CGSize -u _xamarin_simd__CGPoint_objc_msgSendSuper_NVector3_IntPtr_CGSize -u _xamarin_simd__CGPoint_objc_msgSend_stret_NVector3_IntPtr_CGSize -u _xamarin_simd__CGPoint_objc_msgSendSuper_stret_NVector3_IntPtr_CGSize -u _xamarin_simd__NVector3_objc_msgSend_CGPoint_NMatrix4_IntPtr_CGSize -u _xamarin_simd__NVector3_objc_msgSendSuper_CGPoint_NMatrix4_IntPtr_CGSize -u _xamarin_simd__NVector3_objc_msgSend_stret_CGPoint_NMatrix4_IntPtr_CGSize -u _xamarin_simd__NVector3_objc_msgSendSuper_stret_CGPoint_NMatrix4_IntPtr_CGSize -u _xamarin_simd__NMatrix4_objc_msgSend_IntPtr_CGSize_nfloat_nfloat -u _xamarin_simd__NMatrix4_objc_msgSendSuper_IntPtr_CGSize_nfloat_nfloat -u _xamarin_simd__NMatrix4_objc_msgSend_stret_IntPtr_CGSize_nfloat_nfloat -u _xamarin_simd__NMatrix4_objc_msgSendSuper_stret_IntPtr_CGSize_nfloat_nfloat -u _xamarin_simd__NMatrix4_objc_msgSend_IntPtr -u _xamarin_simd__NMatrix4_objc_msgSendSuper_IntPtr -u _xamarin_simd__NMatrix4_objc_msgSend_stret_IntPtr -u _xamarin_simd__NMatrix4_objc_msgSendSuper_stret_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_NMatrix4_NVector3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_NMatrix4_NVector3 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_NMatrix4_NVector3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_NMatrix4_NVector3 -u _xamarin_simd__IntPtr_objc_msgSend_NVector3_NVector3_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_NVector3_NVector3_IntPtr_IntPtr -u _xamarin_simd__NVector3_objc_msgSend_CGPoint_NMatrix4 -u _xamarin_simd__NVector3_objc_msgSendSuper_CGPoint_NMatrix4 -u _xamarin_simd__NVector3_objc_msgSend_stret_CGPoint_NMatrix4 -u _xamarin_simd__NVector3_objc_msgSendSuper_stret_CGPoint_NMatrix4 -u _xamarin_simd__void_objc_msgSend_NMatrix4 -u _xamarin_simd__void_objc_msgSendSuper_NMatrix4 -u _xamarin_simd__void_objc_msgSend_NMatrix4_NVector3_NVector3_IntPtr -u _xamarin_simd__void_objc_msgSendSuper_NMatrix4_NVector3_NVector3_IntPtr -u _xamarin_simd__void_objc_msgSend_NVector3_IntPtr -u _xamarin_simd__void_objc_msgSendSuper_NVector3_IntPtr -u _xamarin_simd__Vector2_objc_msgSend_IntPtr -u _xamarin_simd__Vector2_objc_msgSendSuper_IntPtr -u _xamarin_simd__Vector2_objc_msgSend_stret_IntPtr -u _xamarin_simd__Vector2_objc_msgSendSuper_stret_IntPtr -u _xamarin_simd__NMatrix4x3_objc_msgSend -u _xamarin_simd__NMatrix4x3_objc_msgSendSuper -u _xamarin_simd__NMatrix4x3_objc_msgSend_stret -u _xamarin_simd__NMatrix4x3_objc_msgSendSuper_stret -u _xamarin_IntPtr_objc_msgSend_IntPtr_exception -u _xamarin_IntPtr_objc_msgSendSuper_IntPtr_exception -u _xamarin_IntPtr_objc_msgSend_IntPtr_IntPtr_UIntPtr_exception -u _xamarin_IntPtr_objc_msgSendSuper_IntPtr_IntPtr_UIntPtr_exception -u _xamarin_IntPtr_objc_msgSend_IntPtr_IntPtr_IntPtr_IntPtr_UIntPtr_exception -u _xamarin_IntPtr_objc_msgSendSuper_IntPtr_IntPtr_IntPtr_IntPtr_UIntPtr_exception -u _xamarin_IntPtr_objc_msgSend_IntPtr_UIntPtr_IntPtr_UIntPtr_exception -u _xamarin_IntPtr_objc_msgSendSuper_IntPtr_UIntPtr_IntPtr_UIntPtr_exception -u _xamarin_simd__Vector2_objc_msgSend -u _xamarin_simd__Vector2_objc_msgSendSuper -u _xamarin_simd__Vector2_objc_msgSend_stret -u _xamarin_simd__Vector2_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_Vector2 -u _xamarin_simd__void_objc_msgSendSuper_Vector2 -u _xamarin_simd__Vector3_objc_msgSend -u _xamarin_simd__Vector3_objc_msgSendSuper -u _xamarin_simd__Vector3_objc_msgSend_stret -u _xamarin_simd__Vector3_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_Vector3 -u _xamarin_simd__void_objc_msgSendSuper_Vector3 -u _xamarin_simd__Matrix3_objc_msgSend -u _xamarin_simd__Matrix3_objc_msgSendSuper -u _xamarin_simd__Matrix3_objc_msgSend_stret -u _xamarin_simd__Matrix3_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_Matrix3 -u _xamarin_simd__void_objc_msgSendSuper_Matrix3 -u _xamarin_simd__void_objc_msgSend_NMatrix3 -u _xamarin_simd__void_objc_msgSendSuper_NMatrix3 -u _xamarin_simd__IntPtr_objc_msgSend_Vector2 -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2 -u _xamarin_simd__IntPtr_objc_msgSend_Vector3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3 -u _xamarin_simd__Vector2i_objc_msgSend -u _xamarin_simd__Vector2i_objc_msgSendSuper -u _xamarin_simd__Vector2i_objc_msgSend_stret -u _xamarin_simd__Vector2i_objc_msgSendSuper_stret -u _xamarin_simd__IntPtr_objc_msgSend_Vector2i_int_int_bool -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2i_int_int_bool -u _xamarin_simd__IntPtr_objc_msgSend_Vector2i_int_int_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2i_int_int_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector2i -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2i -u _xamarin_simd__IntPtr_objc_msgSend_float_Vector2_Vector2_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_Vector2_Vector2_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_float_Vector2_Vector2 -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_Vector2_Vector2 -u _xamarin_simd__GKTriangle_objc_msgSend_UIntPtr -u _xamarin_simd__GKTriangle_objc_msgSendSuper_UIntPtr -u _xamarin_simd__GKTriangle_objc_msgSend_stret_UIntPtr -u _xamarin_simd__GKTriangle_objc_msgSendSuper_stret_UIntPtr -u _xamarin_simd__float_objc_msgSend_Vector2 -u _xamarin_simd__float_objc_msgSendSuper_Vector2 -u _xamarin_simd__void_objc_msgSend_Vector3d -u _xamarin_simd__void_objc_msgSendSuper_Vector3d -u _xamarin_simd__Vector2d_objc_msgSend -u _xamarin_simd__Vector2d_objc_msgSendSuper -u _xamarin_simd__Vector2d_objc_msgSend_stret -u _xamarin_simd__Vector2d_objc_msgSendSuper_stret -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Vector2d_Vector2d_Vector2i_bool -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Vector2d_Vector2d_Vector2i_bool -u _xamarin_simd__float_objc_msgSend_Vector2i -u _xamarin_simd__float_objc_msgSendSuper_Vector2i -u _xamarin_simd__void_objc_msgSend_float_Vector2i -u _xamarin_simd__void_objc_msgSendSuper_float_Vector2i -u _xamarin_simd__GKBox_objc_msgSend -u _xamarin_simd__GKBox_objc_msgSendSuper -u _xamarin_simd__GKBox_objc_msgSend_stret -u _xamarin_simd__GKBox_objc_msgSendSuper_stret -u _xamarin_simd__IntPtr_objc_msgSend_GKBox_float -u _xamarin_simd__IntPtr_objc_msgSendSuper_GKBox_float -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Vector3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Vector3 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_GKBox -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_GKBox -u _xamarin_simd__IntPtr_objc_msgSend_GKBox -u _xamarin_simd__IntPtr_objc_msgSendSuper_GKBox -u _xamarin_simd__Vector2_objc_msgSend_UIntPtr -u _xamarin_simd__Vector2_objc_msgSendSuper_UIntPtr -u _xamarin_simd__Vector2_objc_msgSend_stret_UIntPtr -u _xamarin_simd__Vector2_objc_msgSendSuper_stret_UIntPtr -u _xamarin_simd__Vector3_objc_msgSend_UIntPtr -u _xamarin_simd__Vector3_objc_msgSendSuper_UIntPtr -u _xamarin_simd__Vector3_objc_msgSend_stret_UIntPtr -u _xamarin_simd__Vector3_objc_msgSendSuper_stret_UIntPtr -u _xamarin_simd__IntPtr_objc_msgSend_GKQuad_float -u _xamarin_simd__IntPtr_objc_msgSendSuper_GKQuad_float -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Vector2_exception -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Vector2_exception -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_GKQuad -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_GKQuad -u _xamarin_simd__IntPtr_objc_msgSend_GKQuad -u _xamarin_simd__IntPtr_objc_msgSendSuper_GKQuad -u _xamarin_simd__IntPtr_objc_msgSend_Vector2_Vector2_float_exception -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2_Vector2_float_exception -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Vector2_Vector2_exception -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Vector2_Vector2_exception -u _xamarin_simd__IntPtr_objc_msgSend_Vector2_Vector2_exception -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2_Vector2_exception -u _xamarin_simd__GKQuad_objc_msgSend -u _xamarin_simd__GKQuad_objc_msgSendSuper -u _xamarin_simd__GKQuad_objc_msgSend_stret -u _xamarin_simd__GKQuad_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_IntPtr_Vector2_Vector2_IntPtr -u _xamarin_simd__void_objc_msgSendSuper_IntPtr_Vector2_Vector2_IntPtr -u _xamarin_simd__void_objc_msgSend_IntPtr_Vector2_Vector2 -u _xamarin_simd__void_objc_msgSendSuper_IntPtr_Vector2_Vector2 -u _xamarin_simd__MPSAxisAlignedBoundingBox_objc_msgSend -u _xamarin_simd__MPSAxisAlignedBoundingBox_objc_msgSendSuper -u _xamarin_simd__MPSAxisAlignedBoundingBox_objc_msgSend_stret -u _xamarin_simd__MPSAxisAlignedBoundingBox_objc_msgSendSuper_stret -u _xamarin_simd__MPSImageHistogramInfo_objc_msgSend -u _xamarin_simd__MPSImageHistogramInfo_objc_msgSendSuper -u _xamarin_simd__MPSImageHistogramInfo_objc_msgSend_stret -u _xamarin_simd__MPSImageHistogramInfo_objc_msgSendSuper_stret -u _xamarin_simd__Vector4_objc_msgSend -u _xamarin_simd__Vector4_objc_msgSendSuper -u _xamarin_simd__Vector4_objc_msgSend_stret -u _xamarin_simd__Vector4_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_Vector4 -u _xamarin_simd__void_objc_msgSendSuper_Vector4 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_ref_MPSImageHistogramInfo -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_ref_MPSImageHistogramInfo -u _xamarin_simd__void_objc_msgSend_NMatrix4_Double -u _xamarin_simd__void_objc_msgSendSuper_NMatrix4_Double -u _xamarin_simd__void_objc_msgSend_NMatrix4d_Double -u _xamarin_simd__void_objc_msgSendSuper_NMatrix4d_Double -u _xamarin_simd__NMatrix4_objc_msgSend_Double -u _xamarin_simd__NMatrix4_objc_msgSendSuper_Double -u _xamarin_simd__NMatrix4_objc_msgSend_stret_Double -u _xamarin_simd__NMatrix4_objc_msgSendSuper_stret_Double -u _xamarin_simd__NMatrix4d_objc_msgSend_Double -u _xamarin_simd__NMatrix4d_objc_msgSendSuper_Double -u _xamarin_simd__NMatrix4d_objc_msgSend_stret_Double -u _xamarin_simd__NMatrix4d_objc_msgSendSuper_stret_Double -u _xamarin_simd__void_objc_msgSend_Quaternion_Double -u _xamarin_simd__void_objc_msgSendSuper_Quaternion_Double -u _xamarin_simd__void_objc_msgSend_Quaterniond_Double -u _xamarin_simd__void_objc_msgSendSuper_Quaterniond_Double -u _xamarin_simd__Quaternion_objc_msgSend_Double -u _xamarin_simd__Quaternion_objc_msgSendSuper_Double -u _xamarin_simd__Quaternion_objc_msgSend_stret_Double -u _xamarin_simd__Quaternion_objc_msgSendSuper_stret_Double -u _xamarin_simd__Quaterniond_objc_msgSend_Double -u _xamarin_simd__Quaterniond_objc_msgSendSuper_Double -u _xamarin_simd__Quaterniond_objc_msgSend_stret_Double -u _xamarin_simd__Quaterniond_objc_msgSendSuper_stret_Double -u _xamarin_simd__void_objc_msgSend_Vector2_Double -u _xamarin_simd__void_objc_msgSendSuper_Vector2_Double -u _xamarin_simd__void_objc_msgSend_Vector2d_Double -u _xamarin_simd__void_objc_msgSendSuper_Vector2d_Double -u _xamarin_simd__Vector2_objc_msgSend_Double -u _xamarin_simd__Vector2_objc_msgSendSuper_Double -u _xamarin_simd__Vector2_objc_msgSend_stret_Double -u _xamarin_simd__Vector2_objc_msgSendSuper_stret_Double -u _xamarin_simd__Vector2d_objc_msgSend_Double -u _xamarin_simd__Vector2d_objc_msgSendSuper_Double -u _xamarin_simd__Vector2d_objc_msgSend_stret_Double -u _xamarin_simd__Vector2d_objc_msgSendSuper_stret_Double -u _xamarin_simd__void_objc_msgSend_NVector3_Double -u _xamarin_simd__void_objc_msgSendSuper_NVector3_Double -u _xamarin_simd__void_objc_msgSend_NVector3d_Double -u _xamarin_simd__void_objc_msgSendSuper_NVector3d_Double -u _xamarin_simd__NVector3_objc_msgSend_Double -u _xamarin_simd__NVector3_objc_msgSendSuper_Double -u _xamarin_simd__NVector3_objc_msgSend_stret_Double -u _xamarin_simd__NVector3_objc_msgSendSuper_stret_Double -u _xamarin_simd__NVector3d_objc_msgSend_Double -u _xamarin_simd__NVector3d_objc_msgSendSuper_Double -u _xamarin_simd__NVector3d_objc_msgSend_stret_Double -u _xamarin_simd__NVector3d_objc_msgSendSuper_stret_Double -u _xamarin_simd__void_objc_msgSend_Vector4_Double -u _xamarin_simd__void_objc_msgSendSuper_Vector4_Double -u _xamarin_simd__void_objc_msgSend_Vector4d_Double -u _xamarin_simd__void_objc_msgSendSuper_Vector4d_Double -u _xamarin_simd__Vector4_objc_msgSend_Double -u _xamarin_simd__Vector4_objc_msgSendSuper_Double -u _xamarin_simd__Vector4_objc_msgSend_stret_Double -u _xamarin_simd__Vector4_objc_msgSendSuper_stret_Double -u _xamarin_simd__Vector4d_objc_msgSend_Double -u _xamarin_simd__Vector4d_objc_msgSendSuper_Double -u _xamarin_simd__Vector4d_objc_msgSend_stret_Double -u _xamarin_simd__Vector4d_objc_msgSendSuper_stret_Double -u _xamarin_simd__NMatrix4d_objc_msgSend -u _xamarin_simd__NMatrix4d_objc_msgSendSuper -u _xamarin_simd__NMatrix4d_objc_msgSend_stret -u _xamarin_simd__NMatrix4d_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_NMatrix4d -u _xamarin_simd__void_objc_msgSendSuper_NMatrix4d -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSend -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSendSuper -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSend_stret -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_NVector3 -u _xamarin_simd__void_objc_msgSendSuper_NVector3 -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSend_Double -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSendSuper_Double -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSend_stret_Double -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSendSuper_stret_Double -u _xamarin_simd__Matrix4_objc_msgSend -u _xamarin_simd__Matrix4_objc_msgSendSuper -u _xamarin_simd__Matrix4_objc_msgSend_stret -u _xamarin_simd__Matrix4_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_MDLAxisAlignedBoundingBox_bool -u _xamarin_simd__void_objc_msgSendSuper_MDLAxisAlignedBoundingBox_bool -u _xamarin_simd__void_objc_msgSend_Vector3_Vector3 -u _xamarin_simd__void_objc_msgSendSuper_Vector3_Vector3 -u _xamarin_simd__Vector3_objc_msgSend_Vector2i_Vector2i -u _xamarin_simd__Vector3_objc_msgSendSuper_Vector2i_Vector2i -u _xamarin_simd__Vector3_objc_msgSend_stret_Vector2i_Vector2i -u _xamarin_simd__Vector3_objc_msgSendSuper_stret_Vector2i_Vector2i -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_bool_IntPtr_Vector2i_IntPtr_UIntPtr_IntPtr_bool -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_bool_IntPtr_Vector2i_IntPtr_UIntPtr_IntPtr_bool -u _xamarin_simd__IntPtr_objc_msgSend_float_IntPtr_Vector2i_int_IntPtr_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_IntPtr_Vector2i_int_IntPtr_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_float_float_IntPtr_Vector2i -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_float_IntPtr_Vector2i -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_IntPtr_IntPtr_Vector2i -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_IntPtr_IntPtr_Vector2i -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_IntPtr -u _xamarin_simd__void_objc_msgSend_Matrix4 -u _xamarin_simd__void_objc_msgSendSuper_Matrix4 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_UIntPtr_Vector2 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_UIntPtr_Vector2 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_UIntPtr_Vector3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_UIntPtr_Vector3 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_UIntPtr_Vector4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_UIntPtr_Vector4 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_UIntPtr_Matrix4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_UIntPtr_Matrix4 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_UIntPtr_NMatrix4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_UIntPtr_NMatrix4 -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector3i_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector3i_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector2i_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector2i_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector2i_bool_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector2i_bool_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector2i_bool_bool_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector2i_bool_bool_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector2i_int_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector2i_int_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector2i_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector2i_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_bool_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_int_UInt32_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_int_UInt32_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_Vector3i_IntPtr_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_Vector3i_IntPtr_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector2_Vector2i_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2_Vector2i_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_Vector3_UIntPtr_UIntPtr_IntPtr_bool_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector3_UIntPtr_UIntPtr_IntPtr_bool_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_float_Vector2_UIntPtr_UIntPtr_IntPtr_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_Vector2_UIntPtr_UIntPtr_IntPtr_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_float_Vector2_UIntPtr_UIntPtr_UIntPtr_IntPtr_bool_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_Vector2_UIntPtr_UIntPtr_UIntPtr_IntPtr_bool_IntPtr -u _xamarin_simd__bool_objc_msgSend_Vector2i_IntPtr_float_IntPtr_IntPtr_IntPtr -u _xamarin_simd__bool_objc_msgSendSuper_Vector2i_IntPtr_float_IntPtr_IntPtr_IntPtr -u _xamarin_simd__bool_objc_msgSend_Vector2i_IntPtr_IntPtr_IntPtr_IntPtr -u _xamarin_simd__bool_objc_msgSendSuper_Vector2i_IntPtr_IntPtr_IntPtr_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_float_IntPtr_Vector2i_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_IntPtr_Vector2i_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_float_IntPtr_Vector2i_int_IntPtr_bool -u _xamarin_simd__IntPtr_objc_msgSendSuper_float_IntPtr_Vector2i_int_IntPtr_bool -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_IntPtr_Vector2i_float_float_float_float -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_IntPtr_Vector2i_float_float_float_float -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_IntPtr_Vector2i_float_float_float_float_float -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_IntPtr_Vector2i_float_float_float_float_float -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_IntPtr_Vector2i -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_IntPtr_Vector2i -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_IntPtr_Vector2i_float -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_IntPtr_Vector2i_float -u _xamarin_simd__IntPtr_objc_msgSend_Matrix4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_Matrix4 -u _xamarin_simd__IntPtr_objc_msgSend_Matrix4_bool -u _xamarin_simd__IntPtr_objc_msgSendSuper_Matrix4_bool -u _xamarin_simd__IntPtr_objc_msgSend_NMatrix4_bool -u _xamarin_simd__IntPtr_objc_msgSendSuper_NMatrix4_bool -u _xamarin_simd__Vector3_objc_msgSend_Double -u _xamarin_simd__Vector3_objc_msgSendSuper_Double -u _xamarin_simd__Vector3_objc_msgSend_stret_Double -u _xamarin_simd__Vector3_objc_msgSendSuper_stret_Double -u _xamarin_simd__Matrix4_objc_msgSend_Double -u _xamarin_simd__Matrix4_objc_msgSendSuper_Double -u _xamarin_simd__Matrix4_objc_msgSend_stret_Double -u _xamarin_simd__Matrix4_objc_msgSendSuper_stret_Double -u _xamarin_simd__void_objc_msgSend_Vector3_Double -u _xamarin_simd__void_objc_msgSendSuper_Vector3_Double -u _xamarin_simd__void_objc_msgSend_Matrix4_Double -u _xamarin_simd__void_objc_msgSendSuper_Matrix4_Double -u _xamarin_simd__Matrix4_objc_msgSend_IntPtr_Double -u _xamarin_simd__Matrix4_objc_msgSendSuper_IntPtr_Double -u _xamarin_simd__Matrix4_objc_msgSend_stret_IntPtr_Double -u _xamarin_simd__Matrix4_objc_msgSendSuper_stret_IntPtr_Double -u _xamarin_simd__MDLVoxelIndexExtent2_objc_msgSend -u _xamarin_simd__MDLVoxelIndexExtent2_objc_msgSendSuper -u _xamarin_simd__MDLVoxelIndexExtent2_objc_msgSend_stret -u _xamarin_simd__MDLVoxelIndexExtent2_objc_msgSendSuper_stret -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_MDLAxisAlignedBoundingBox_float -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_MDLAxisAlignedBoundingBox_float -u _xamarin_simd__bool_objc_msgSend_Vector4i_bool_bool_bool_bool -u _xamarin_simd__bool_objc_msgSendSuper_Vector4i_bool_bool_bool_bool -u _xamarin_simd__void_objc_msgSend_Vector4i -u _xamarin_simd__void_objc_msgSendSuper_Vector4i -u _xamarin_simd__IntPtr_objc_msgSend_MDLVoxelIndexExtent2 -u _xamarin_simd__IntPtr_objc_msgSendSuper_MDLVoxelIndexExtent2 -u _xamarin_simd__Vector4i_objc_msgSend_Vector3 -u _xamarin_simd__Vector4i_objc_msgSendSuper_Vector3 -u _xamarin_simd__Vector4i_objc_msgSend_stret_Vector3 -u _xamarin_simd__Vector4i_objc_msgSendSuper_stret_Vector3 -u _xamarin_simd__Vector3_objc_msgSend_Vector4i -u _xamarin_simd__Vector3_objc_msgSendSuper_Vector4i -u _xamarin_simd__Vector3_objc_msgSend_stret_Vector4i -u _xamarin_simd__Vector3_objc_msgSendSuper_stret_Vector4i -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSend_Vector4i -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSendSuper_Vector4i -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSend_stret_Vector4i -u _xamarin_simd__MDLAxisAlignedBoundingBox_objc_msgSendSuper_stret_Vector4i -u _xamarin_simd__Quaternion_objc_msgSend -u _xamarin_simd__Quaternion_objc_msgSendSuper -u _xamarin_simd__Quaternion_objc_msgSend_stret -u _xamarin_simd__Quaternion_objc_msgSendSuper_stret -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Quaternion_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Quaternion_IntPtr -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Quaternion -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Quaternion -u _xamarin_simd__IntPtr_objc_msgSend_Vector2d_IntPtr -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector2d_IntPtr -u _xamarin_simd__void_objc_msgSend_Vector2d -u _xamarin_simd__void_objc_msgSendSuper_Vector2d -u _xamarin_simd__Vector3_objc_msgSend_Vector3 -u _xamarin_simd__Vector3_objc_msgSendSuper_Vector3 -u _xamarin_simd__Vector3_objc_msgSend_stret_Vector3 -u _xamarin_simd__Vector3_objc_msgSendSuper_stret_Vector3 -u _xamarin_simd__IntPtr_objc_msgSend_Vector4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_Vector4 -u _xamarin_vector_float3__Vector4_objc_msgSend -u _xamarin_vector_float3__Vector4_objc_msgSendSuper -u _xamarin_vector_float3__Vector4_objc_msgSend_stret -u _xamarin_vector_float3__Vector4_objc_msgSendSuper_stret -u _xamarin_vector_float3__void_objc_msgSend_Vector4 -u _xamarin_vector_float3__void_objc_msgSendSuper_Vector4 -u _xamarin_simd__void_objc_msgSend_Quaternion -u _xamarin_simd__void_objc_msgSendSuper_Quaternion -u _xamarin_simd__NMatrix2_objc_msgSend -u _xamarin_simd__NMatrix2_objc_msgSendSuper -u _xamarin_simd__NMatrix2_objc_msgSend_stret -u _xamarin_simd__NMatrix2_objc_msgSendSuper_stret -u _xamarin_simd__void_objc_msgSend_NMatrix2 -u _xamarin_simd__void_objc_msgSendSuper_NMatrix2 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Vector4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Vector4 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Matrix2 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Matrix2 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_NMatrix2 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_NMatrix2 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Matrix3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Matrix3 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_NMatrix3 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_NMatrix3 -u _xamarin_simd__IntPtr_objc_msgSend_IntPtr_Matrix4 -u _xamarin_simd__IntPtr_objc_msgSendSuper_IntPtr_Matrix4 -u _NetSecurityNative_ReleaseGssBuffer -u _NetSecurityNative_DisplayMinorStatus -u _NetSecurityNative_DisplayMajorStatus -u _NetSecurityNative_ImportUserName -u _NetSecurityNative_ImportPrincipalName -u _NetSecurityNative_ReleaseName -u _NetSecurityNative_InitiateCredSpNego -u _NetSecurityNative_InitiateCredWithPassword -u _NetSecurityNative_ReleaseCred -u _NetSecurityNative_InitSecContext -u _NetSecurityNative_InitSecContextEx -u _NetSecurityNative_AcceptSecContext -u _NetSecurityNative_DeleteSecContext -u _NetSecurityNative_GetUser -u _NetSecurityNative_Wrap -u _NetSecurityNative_Unwrap -u _SystemNative_GetDomainSocketSizes -u _BrotliDecoderCreateInstance -u _BrotliDecoderDecompressStream -u _BrotliDecoderDecompress -u _BrotliDecoderDestroyInstance -u _BrotliDecoderIsFinished -u _BrotliEncoderCreateInstance -u _BrotliEncoderSetParameter -u _BrotliEncoderCompressStream -u _BrotliEncoderHasMoreOutput -u _BrotliEncoderDestroyInstance -u _BrotliEncoderCompress -u _OBJC_CLASS_$_OneSignal -u _OBJC_CLASS_$_OneSignalWrapper -u _OBJC_CLASS_$_OSDisplayableNotification -u _OBJC_CLASS_$_OSInAppMessage -u _OBJC_CLASS_$_OSInAppMessageClickEvent -u _OBJC_CLASS_$_OSInAppMessageClickResult -u _OBJC_CLASS_$_OSInAppMessageDidDismissEvent -u _OBJC_CLASS_$_OSInAppMessageDidDisplayEvent -u _OBJC_CLASS_$_OSInAppMessageOutcome -u _OBJC_CLASS_$_OSInAppMessageTag -u _OBJC_CLASS_$_OSInAppMessageWillDismissEvent -u _OBJC_CLASS_$_OSInAppMessageWillDisplayEvent -u _OBJC_CLASS_$_OSNotification -u _OBJC_CLASS_$_OSNotificationClickEvent -u _OBJC_CLASS_$_OSNotificationClickResult -u _OBJC_CLASS_$_OSNotificationWillDisplayEvent -u _OBJC_CLASS_$__TtC13OneSignalUser30OSPushSubscriptionChangedState -u _OBJC_CLASS_$__TtC13OneSignalUser23OSPushSubscriptionState -u _mono_pmip -u _xamarin_dyn_objc_msgSend -u _xamarin_dyn_objc_msgSendSuper -u _xamarin_dyn_objc_msgSend_stret -u _xamarin_dyn_objc_msgSendSuper_stret
    Undefined symbols for architecture x86_64:
      "_OBJC_CLASS_$_OSInAppMessage", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageClickEvent", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageClickResult", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageDidDismissEvent", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageDidDisplayEvent", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageTag", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageWillDismissEvent", referenced from:
         -u command line option
      "_OBJC_CLASS_$_OSInAppMessageWillDisplayEvent", referenced from:
         -u command line option
    ld: symbol(s) not found for architecture x86_64
    clang : error : linker command failed with exit code 1 (use -v to see invocation)
     
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessage. The symbol '_OBJC_CLASS_$_OSInAppMessage' could not be found in any of the libraries or frameworks linked with your application.
    		
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessageClickEvent. The symbol '_OBJC_CLASS_$_OSInAppMessageClickEvent' could not be found in any of the libraries or frameworks linked with your application.
    		
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessageClickResult. The symbol '_OBJC_CLASS_$_OSInAppMessageClickResult' could not be found in any of the libraries or frameworks linked with your application.
    		
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessageDidDismissEvent. The symbol '_OBJC_CLASS_$_OSInAppMessageDidDismissEvent' could not be found in any of the libraries or frameworks linked with your application.
    		
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessageDidDisplayEvent. The symbol '_OBJC_CLASS_$_OSInAppMessageDidDisplayEvent' could not be found in any of the libraries or frameworks linked with your application.
    		
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessageTag. The symbol '_OBJC_CLASS_$_OSInAppMessageTag' could not be found in any of the libraries or frameworks linked with your application.
    		
    MTOUCH : error MT5211: Native linking failed, undefined Objective-C class: OSInAppMessageWillDismissEvent. The symbol '_OBJC_CLASS_$_OSInAppMessageWillDismissEvent' could not be found in any of the libraries or frameworks linked with your application.


How come? and why don't I have problems with OnesignalSdk.Xamarin? Maybe OnesignalSdk.Dotnet is not compatible with xamarin projects and I should use Net Maui?

Thank you

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/OneSignalSDK.DotNet.iOS.Binding/OneSignalSDK.DotNet.iOS.Binding.csproj
+++ b/OneSignalSDK.DotNet.iOS.Binding/OneSignalSDK.DotNet.iOS.Binding.csproj
@@ -85,5 +85,17 @@
       <ForceLoad>True</ForceLoad>
       <Frameworks>OneSignalCore OneSignalNotifications OneSignalOSCore</Frameworks>
     </NativeReference>
+    <NativeReference Include="OneSignalLocation.xcframework">
+      <Kind>Framework</Kind>
+      <SmartLink>False</SmartLink>
+      <ForceLoad>True</ForceLoad>
+      <Frameworks>OneSignalCore OneSignalNotifications OneSignalOSCore OneSignalUser UIKit</Frameworks>
+    </NativeReference>
+    <NativeReference Include="OneSignalInAppMessages.xcframework">
+      <Kind>Framework</Kind>
+      <SmartLink>False</SmartLink>
+      <ForceLoad>True</ForceLoad>
+      <Frameworks>OneSignalCore OneSignalNotifications OneSignalOSCore OneSignalUser OneSignalOutcomes UIKit WebKit CoreGraphics</Frameworks>
+    </NativeReference>
   </ItemGroup>
 </Project>

